### DISCUSSION TITLE
Select a cell from other document

### DISCUSSION THREAD

**Discussion by urbi3** [PHPOffice/PhpSpreadsheet#3588] [Q&A] [answered]
I started using Gnumeric. How can I select a cell from another document? In Excel I could select a cell from another opened document like selecting a cell on the same spreadsheet with the mouse through the data entry area. In Gnumeric it doesn't seem to work this simple. How can I manage this task?

**Comment by oleibman** [comment, marked as answer]
It's probably best to ask this in a Gnumeric forum. We try to support Gnumeric files, but there may certainly be gaps in our knowledge. As far as I can tell, what you are looking for does not exist in Gnumeric. For example, see http://www.fifi.org/doc/gnumeric-doc/html/C/cell-referencing.html#AEN1580, which doesn't even mention this possibility. Also, when I use this construction in an Xlsx spreadsheet, and load that spreadsheet in Gnumeric, it treats it as an invalid reference (`#REF!`).

**Comment by oleibman** [comment]
No update in a year and a half, no reason to think the suggested answer is wrong. Marking it as answer.
